executor/prepared.go

```diff
@@ -322,6 +325,7 @@ func CompileExecutePreparedStmt(ctx context.Context, sctx sessionctx.Context,
 		stmt.Text = preparedObj.PreparedAst.Stmt.Text()
 		stmtCtx.OriginalSQL = stmt.Text
 		stmtCtx.InitSQLDigest(preparedObj.NormalizedSQL, preparedObj.SQLDigest)
+		sctx.GetSessionVars().StmtCtx.OriginalSQL = stmt.Text
```

Comment:
is this a bugfix?

Reply:
It seems that someone changed the code here, and I didn't merge on time.